web/src/main/java/org/cbioportal/genome_nexus/web/error/GlobalExceptionHandler.java

```diff
@@ -68,10 +68,4 @@
     {
         return new ResponseEntity<>(new ErrorResponse(ex.getResponseBody()), ex.getStatusCode());
     }
-
-    @ExceptionHandler(IsoformOverrideNotFoundException.class)
-    public ResponseEntity<ErrorResponse> handleIsoformOverrideNotFound(IsoformOverrideNotFoundException ex)
-    {
-        return new ResponseEntity<>(new ErrorResponse(ex.getLocalizedMessage()), HttpStatus.NOT_FOUND);
```

Comment:
see my previous comment. If we want to leave the controller so that a custom message goes to the community with a http status of HttpStatus.GONE we could modify this class to generate such a response (maybe with a new exception type or using UnsupportedOperation)

Reply:
resolved .. not necessary